spec/generators/dashboard_generator_spec.rb

```diff
@@ -61,6 +61,31 @@ class Foo < ApplicationRecord
         end
       end
 
+      it "sorts the attributes" do
+        begin
+          ActiveRecord::Schema.define do
+            create_table(:foos) do |t|
+              t.string :col_2
+              t.string :col_1
+              t.string :col_3
+              t.timestamps
+            end
+          end
+
+          class Foo < ApplicationRecord
+            reset_column_information
+          end
+
+          run_generator ["foo"]
+          load file("app/dashboards/foo_dashboard.rb")
+          attrs = FooDashboard::ATTRIBUTE_TYPES.keys
+
+          expect(attrs).to eq(%i[id col_1 col_2 col_3 created_at updated_at])
```

Comment:
Companion to my comment above about a primary key:

```suggestion
          expect(attrs).to eq(%i[code col_1 col_2 col_3 created_at updated_at])
```

Reply:
Updated.